docs/vsphere-prerequisites.md

```diff
@@ -1,27 +1,34 @@
 # vSphere pre-requisites
+
+In order to use the Windows MachineConfig Operator to successfully create a Windows node, you will first need to create a a Windows "golden image" VM. These steps are  things that must be done, and things the Operator expects to be there when the Windows golden image VM is cloned.
```

Comment:
In your 2nd sentence, I suggest saying something like `These steps are an overview of the required configuration of the golden image.` I am sure @codyhoag can wordsmith it better than both of us :-)

Reply:
Updated!